src/constants.ts

```diff
@@ -71,3 +71,12 @@ export type QuickPicksCache = { cache: QuickPickItem[], next: string | null };
 
 // Originally from the Docker extension: https://github.com/microsoft/vscode-docker/blob/main/src/constants.ts
 export const DOCKERFILE_GLOB_PATTERN = '**/{*.[dD][oO][cC][kK][eE][rR][fF][iI][lL][eE],[dD][oO][cC][kK][eE][rR][fF][iI][lL][eE],[dD][oO][cC][kK][eE][rR][fF][iI][lL][eE].*}';
+
+export const unsavedChangesTrueContextValue: string = 'unsavedChanges:true';
```

Comment:
Seems like it's only used in `ContainerAppItem` right now. Is that going to change or was there a different reason for having it in this file?

Reply:
I think for my purposes so far, it will only be used in `ContainerAppItem` so I'm happy to move it back.  At the time I was thinking way down the line and it may end up being used by other items, but that's probably in a future future PR